AlpsSDK/AlpsManager/AlpsManager.swift

```diff
@@ -30,7 +30,7 @@ public class AlpsManager: MatchMonitorDelegate, ContextManagerDelegate {
     
     public lazy var contextManager = ContextManager(delegate: self)
     public lazy var matchMonitor = MatchMonitor(delegate: self)
-    public let remoteNotificationManager: RemoteNotificationManager = RemoteNotificationManager()
+    public var remoteNotificationManager: RemoteNotificationManager!
```

Comment:
Reversed it back to `var`, I am having issue with `self`when it is `let`

Reply:
I think we can change it to let without `!`. Just make sure you init it in evey AlpsManager's `init()`